controllers/controller.js

```diff
@@ -61,3 +63,13 @@ function postComment(request, response, next) {
     })
     .catch(next);
 }
+function patchArticle(request, response, next) {
+  const ArticleId = request.params.article_id;
```

Comment:
this should not be capitalised as this indicates to JS that this variable contains a class, which it does not.